### ISSUE TITLE
Pilocarpine (Medication Used to Lower Intraocular Pressure)

### ISSUE THREAD

**Issue by Sukramg** [monarch-initiative/MAxO#554]
% 600510 OCULAR PIGMENT DISPERSION WITH OR WITHOUT GLAUCOMA; OPDG

Weinreb RN, Aung T, Medeiros FA. The pathophysiology and treatment of glaucoma: a review. JAMA. 2014 May 14;311(18):1901-11. doi: 10.1001/jama.2014.3192. PMID: 24825645; PMCID: PMC4523637.
https://pubmed.ncbi.nlm.nih.gov/24825645/

Ocular pigment dispersion syndrome (PDS) is a condition where pigment granules from the iris rub off onto the lens and other structures in the eye, leading to increased intraocular pressure and a type of glaucoma called pigmentary glaucoma. Pilocarpine is a medication that can be used to treat ocular pigment dispersion glaucoma.

Pilocarpine is a type of cholinergic medication that works by stimulating the muscles in the eye to constrict, which increases the outflow of aqueous humor, the fluid that circulates inside the eye. By increasing the outflow of aqueous humor, pilocarpine can help to lower intraocular pressure, which is a key factor in glaucoma.

Pilocarpine is typically administered as an eye drop, and the frequency and dosage of the drops can vary depending on the severity of the condition and the patient's response to the medication. However, pilocarpine can have side effects such as blurred vision, headache, and eye irritation, and it may not be appropriate for all patients.

In addition to pilocarpine, other medications and treatments may be used to manage ocular pigment dispersion glaucoma, such as topical and oral medications to lower intraocular pressure, laser trabeculoplasty, and conventional glaucoma surgery.

**Comment by pnrobinson** [comment]
Could we make a term with the medication class?
Would this be 
Cholinergic eyedrop medication?
@LCCarmody  Can we start a subhierarchy for eyedrops?

**Comment by LCCarmody** [comment]
Yes, we likely need that. I was wondering if there needs to be a small branch with route of administration. I.e., oral, IV, topical, injection (muscle and subcutaneous?). Not sure how specific to gol

**Comment by LCCarmody** [comment]
added as child under antiglaucoma agent therapy and cholinergic agonist therapy
